contracts/ActivePool.sol

```diff
@@ -186,6 +207,20 @@ contract ActivePool is
 		callerIsBorrowerOperationOrDefaultPool
 	{
 		assetsBalance[_asset] = assetsBalance[_asset].add(_amount);
+
+		if (address(collStakingManager) != address(0) && collStakingManager.isSupportedAsset(_asset)) {
+			bytes memory data = abi.encodeWithSignature("stakeCollaterals(address,uint256)", _asset, _amount);
+
+			if (IERC20Upgradeable(_asset).allowance(address(this), address(collStakingManager)) < _amount) {
+				IERC20Upgradeable(_asset).safeApprove(address(collStakingManager), type(uint256).max);
+			}
+
+			(bool bSuccess, ) = address(collStakingManager).call{value: 0}(data);
```

Comment:
Is there a specific reason why you are doing a bytedata call and not a function call?

```
try collStakingManager.stakeCollaterals(.. , ..) {
} catch () {}
```